makefile

```diff
@@ -1,17 +1,44 @@
-migrations:
-	python3 backend/manage.py makemigrations
+# Backend commands
+VENV := backend/venv
+PYTHON := $(VENV)/bin/python
+PIP := $(VENV)/bin/pip
```

Comment:
is pip aliased for pip3 or how to confirm it? @DevilsAutumn

Reply:
when a virtual environment is created using python3(or python2) , it adds both `pip` and `pip3`(or `pip2`) inside that venv so it doesn't matter if you run `pip` or `pip3` when your venv is active. But if you run `pip` outside your venv, it will say `command not found` but `pip3`(or `pip2`) will still work.

Reply:
got it, thanks.